packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1996,10 +2012,10 @@ describe('plugin-meetings', () => {
                   assert.notCalled(locusMediaRequestStub);
                 }
                 if (isMultistream) {
-                  assert.calledOnceWithExactly(fakeMultistreamRoapMediaConnection.publishTrack, fakeMicrophoneTrack);
+                  assert.calledOnceWithExactly(meeting.sendSlotManager.getSlot(MediaType.AudioMain).publishStream, fakeMicrophoneStream);
                 } else {
                   assert.calledOnceWithExactly(fakeRoapMediaConnection.update, {
-                    localTracks: { audio: webrtcAudioTrack, video: null, screenShareVideo: null },
+                    localStreams: { audio: fakeMicrophoneStream, video: null, screenShareVideo: null },
```

Comment:
this doesn't look right. RoapMediaConnection API takes webrtc tracks not streams (same in line 2060)